src/Parser/DeathKnight/Blood/Modules/Talents/Voracious.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import { formatPercentage } from 'common/format';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+
+class Voracious extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+  };
+
+  on_initialized() {
+    this.active = this.combatants.selected.hasTalent(SPELLS.VORACIOUS_TALENT.id);
+  }
+
+  get uptime() {
+    return this.combatants.selected.getBuffUptime(SPELLS.VORACIOUS.id) / this.owner.fightDuration;
+  }
+
+  get uptimeSuggestionThresholds() {
+    return {
+      actual: this.uptime,
+      isLessThan: {
+        minor: 0.95,
+        average: 0.9,
+        major: .8,
+      },
+      style: 'percentage',
+    };
+  }
+
+  statistic() {
+    return (
+      <StatisticBox
+        icon={<SpellIcon id={SPELLS.VORACIOUS_TALENT.id} />}
+        value={`${formatPercentage(this.uptime)} %`}
+        label="Voracious uptime"
+      />
+    );
+  }
+  statisticOrder = STATISTIC_ORDER.CORE(1);
```

Comment:
You don't want to change this so it doesn't mess with the layout changes you made in the last PR?

Reply:
that was for live, i will worry about this BFA branch when the time comes lol